src/components/Card/Card.module.scss

```
@@ -0,0 +1,49 @@
+@use "../../styles/variables" as *;
+
+.card {
+  width: 360px;
+  height: 100%;
+  border-radius: 10px;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+}
+
+.card:hover {
+  box-shadow: 0 4px 10px rgba(0, 0, 0, 0.25);
+}
```

используй возможности scss
```suggestion
.card {
  width: 360px;
  height: 100%;
  border-radius: 10px;
  overflow: hidden;
  display: flex;
  flex-direction: column;
  
  &:hover {
    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.25);
  }
}
```

---

Поправил